intro-Linux/ch_uploading_files.tex

```diff
@@ -1,26 +1,34 @@
 \chapter{Uploading/downloading/editing files}
 
-\subsection{Uploading/downloading files}
+\section{Uploading/downloading files}
+\label{sec:uploading-files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in chapter 3 of the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
+\hypertarget{sec:dos2unix}{}
 
 \begin{prompt}
 %\shellcmd{dos2unix filename}
 \end{prompt}
 
-as this will fix any problems with windows / unix conversion.
-
+as this will fix any problems with Windows/Unix conversion.
+\fi
 \section{Symlinks for data/scratch}
 
 As we end up in the home directory when connecting, it would be convenient if we
-could access our data and vo storage. To facilitate this we shall create
-symlinks to them in our home directory.
+could access our data and VO storage. To facilitate this we will create
+symlinks to them in our home directory. This will create 4 symbolic links
+\ifwindows
+(they're like ``shortcuts'' on your desktop and they look like directories in WinSCP)
+\else
+(they're like ``shortcuts'' on your desktop)
```

Comment:
There's really no way to avoid duplicating this? :(

Reply:
There probably is, but I couldn't find it in 5 minutes, and by then I felt like I already wasted too much time just to deduplicate it.